Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h

```diff
@@ -659,6 +661,8 @@ class GraphicsLayerCA : public GraphicsLayer, public PlatformCALayerClient {
     std::unique_ptr<DisplayList::InMemoryDisplayList> m_displayList;
 
     float m_contentsScaleLimitingFactor { 1 };
+    bool m_shouldUpdateRootRelativeScaleFactor { false };
```

Comment:
Packing is non-optimal here (you waste 3 bytes).

Reply:
Hmm, do we support bool m_foo : 1 { false } already? I'll check if we have that C++20 feature at hand.

Reply:
Oh it's already used widely, nice.

```
diff --git a/Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h b/Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h
index 367ad35c845d..a6df8d50c480 100644
--- a/Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h
+++ b/Source/WebCore/platform/graphics/ca/GraphicsLayerCA.h
@@ -661,12 +661,11 @@ private:
     std::unique_ptr<DisplayList::InMemoryDisplayList> m_displayList;
 
     float m_contentsScaleLimitingFactor { 1 };
-    bool m_shouldUpdateRootRelativeScaleFactor { false };
     float m_rootRelativeScaleFactor { 1.0f };
 
     ContentsLayerPurpose m_contentsLayerPurpose { ContentsLayerPurpose::None };
-    bool m_isCommittingChanges { false };
-
+    bool m_isCommittingChanges : 1 { false };
+    bool m_shouldUpdateRootRelativeScaleFactor : 1 { false };
     bool m_needsFullRepaint : 1;
     bool m_allowsBackingStoreDetaching : 1;
     bool m_intersectsCoverageRect : 1;
```

This removes wasting space here and in another place + sorted for minimal padding.

Reply:
```
./platform/graphics/ca/GraphicsLayerCA.cpp:1954:41: error: non-const reference cannot bind to bit-field 'm_isCommittingChanges'
    SetForScope committingChangesChange(m_isCommittingChanges, true);
```

Oops, m_isCommittingChanges cannot carry a bit field suffix. Will fix and land.